tests/auth_tests/test_models.py

```diff
@@ -189,6 +191,17 @@ def forwards(apps, schema_editor):
         user = User.objects.get(username='user1')
         self.assertTrue(user.check_password('secure'))
 
+    def test_normalize_email_deprecation(self):
+        msg = 'BaseUserManager.normalize_email() is deprecated in favor of '
+        msg += 'AbstractBaseUser.normalize_email() and will be removed in Django 4.1.'
+        with self.assertWarnsMessage(RemovedInDjango41Warning, msg):
```

Comment:
Other deprecation tests I've seen use `assertRaisesMessage` 🤔

Reply:
I did a quick search, and it appeared to be about 50-50 `assertRaisesMessage` / `assertWarnsMessage` for deprecations.

Reply:
Happened to check the patch submission checklist, and the deprecation guide uses `assertWarnsMessage` FWIW.

Reply:
Ok. That's interesting. Let's see what the mergers have to say!